modules/extensions-support/src/main/java/org/mule/module/extension/internal/introspection/ImmutableExtensionModel.java

```diff
@@ -67,6 +71,7 @@ protected ImmutableExtensionModel(String name,
 
         checkArgument(!StringUtils.isBlank(version), "version cannot be blank");
         this.version = version;
+        this.vendor = vendor;
```

Comment:
Validating this property it's not entire clear, meanwhile using annotations "you can avoid the vendor configuration", when you use the fluent api, forcing the configuration of the vendor it's not intuitive to think that **No Vendor** means `.withVendor("")`, already exist the validation in `DefaultExtensionManager` which checks if it is null, and in this case, register the extension with a blank vendor.